VMBackup/main/fsfreezer.py

```diff
@@ -248,10 +238,23 @@ def freeze_safe(self,timeout):
             self.logger.log(error_msg, True, 'Error')
         return freeze_result,timedout
 
+    def releaseFileLock(self):
+        if (self.isAquireLockSucceeded == True):
+            try:
+                fcntl.lockf(self.file, fcntl.LOCK_UN)
+                self.file.close()
+            except:
+                pass
+        try:
+            os.remove("/etc/azure/MicrosoftRecoverySvcsSafeFreezeLock/SafeFreezeLockFile")
+        except:
+            pass
+
     def thaw_safe(self):
         thaw_result = FreezeResult()
         unable_to_sleep = False
         if(self.skip_freeze == True):
+            self.releaseFileLock()
```

Comment:
if we have already skipped the freeze, do we need to call release here ?

Reply:
yes. removed it.